bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/i18n/LocaleProvider.java

```diff
@@ -25,4 +28,10 @@
      */
     Locale getLocale();
 
+    /**
+     * Provides access to the location of the installation
+     *
+     * @return location of the current installation or null if the location is not set
+     */
+    PointType getLocation();
```

Comment:
> So, it is not about that the implementation could be done in one class but I don't need an implementation of a transaltion mechanism just because I would like to know the location or locale of a system.

That is in fact the core of it all. Maybe we should split up the interface in two parts (LocaleProvider && LocationProvider) but keep the implementation of the configuration of I18n related stuff together.